packages/now-cli/test/dev/integration.js

```diff
@@ -656,6 +656,7 @@ test(
   testFixtureStdio('public-and-api', async testPath => {
     await testPath(200, '/', 'This is the home page');
     await testPath(200, '/about.html', 'This is the about page');
+    await testPath(200, '/.well-known/humans.txt', 'We come in peace');
```

Comment:
Should we also add a test for e.g. `.env`?

Reply:
I wanted to add a test for a real file that a user expects to be served like in #4475 so that's why its `.well-known/humans.txt`.

Do users want to serve the `.env` file? Also, what should the content-type be?